app/modules/integrations/integrations_service.py

```
@@ -2329,3 +2401,351 @@ async def deactivate_jira_integrations_for_user(self, user_id: str) -> int:
+                },
+            )
+
+        if response.status_code != 200:
+            logging.error(
+                "Failed to fetch Confluence spaces (%s): %s",
+                response.status_code,
+                response.text,
+            )
+            raise Exception(
+                f"Failed to fetch Confluence spaces: {response.status_code}"
+            )
```

Ruff (0.14.4)

Create your own exception

(TRY002)